export_results.py

```diff
@@ -495,29 +503,200 @@ def export_updated_phenos(num_pops=None):
                        skip_binary_eur=False)
 
 
-def make_pheno_manifest(export=True):    
+def make_per_population_n_cases():
+    """
+    This function computes per-population and per-sex case count for each phenotype.
+    """
+    path_loc = f'{bucket}/combined_results/per_pop_per_sex_case_counts.ht'
+    ht_saige = hl.read_matrix_table(get_variant_results_path('full')).cols()
+    ht_saige = ht_saige.explode('pheno_data')
+    ht_saige = ht_saige.annotate(pop = ht_saige.pheno_data.pop).key_by(*PHENO_KEY_FIELDS, 'pop')
+
+    if hl.hadoop_exists(path_loc):
```

Comment:
Check for a `_SUCCESS` file - slightly more reliable

Reply:
done